### PR TITLE
Fix issue 653

### PR DESCRIPTION
**Pull request by aorlati**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -97,6 +97,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/
 
 ## [next release]
 ## Added
+    issue #653 - added a couple of scriptsto ease vnc conection from remote a file copying. The scripts support
```
Comment (giuseppe-carboni) [inline CHANGELOG.md:100]: There's a typo, `scriptsto` should be splitted

File: Medicina/Misc/MedScripts/src/med-vnc [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+#echo $viewonly
+#echo $vnc_avail
+
+if [[ "$local_connection" = "TRUE" ]]; then
+    echo "Connecting from local area...."
+    #ssh -N -f -L $local_port:192.168.1.99:$remote_port $role@192.167.189.98
+else
+    echo "Connecting from wide area....."
+fi
+ssh -N -f -p $ssh_port -L $local_port:192.167.189.98:$remote_port $role@$gatein
```
Comment (giuseppe-carboni) [inline Medicina/Misc/MedScripts/src/med-vnc:158]: Shouldn't this be inside the previous `else` block? If I remember correctly, the vnc session needs a ssh tunnel only when connecting from outside (in LAN the vnc should be directly accessible, at least with the latest updates on the vnc servers). I also understand that this script will be used when connecting from the inside by connecting to the gateway and not directly to the desired machine. In that case this line is correct to be here.

Comment (aorlati) [reply Medicina/Misc/MedScripts/src/med-vnc:158]: That's correct the idea is to directly connecting to discos when insider the local network, but this needs a configuration in the proxy machine that cannot be done right now (until the old escs system is decommissioned), until then we go through the gatein also from LAN. I also received some more comments on theses script si i'm going to have some changes to cope with them. I'll send un update soon.

**Comment by aorlati** [comment]
@giuseppe-carboni please check the new version

**Review by giuseppe-carboni — APPROVED** [review]

**Pull request merged**